lib/crypto/crypto_keys.c

```diff
@@ -124,8 +126,26 @@ crypto_keys_init(void)
 {
 	uint8_t randbuf[RANDBUFLEN];
 
-	/* No keys yet. */
-	memset(&keycache, 0, sizeof(keycache));
+	/* No keys yet.  Set pointers to NULL for non-0 NULL platforms. */
```

Comment:
No need to change the comment here.  If you do want to change the comment, I'd go with "memset is insufficient since NULL is not required to be represented in memory by zeroes".